Analytics/Helpers/SEGBluetooth.m

```diff
@@ -28,7 +28,10 @@ - (id)init {
 
   _queue = dispatch_queue_create("io.segment.bluetooth.queue", NULL);
 
-  if (&CBCentralManagerOptionShowPowerAlertKey != NO) {
+  // Reading the address inline fails on iOS 6, must use a temporary
+  // variable to compare.
+  NSInteger addr = &CBCentralManagerOptionShowPowerAlertKey;
+  if (addr != 0) {
```

Comment:
Sorry I meant why not change the temporary variable as a Boolean instead of messing with a pointer, since that's what CBCentralManagerOptionShowPowerAlertKey is: 

![CBCentralManagerOptionShowPowerAlertKey](https://cldup.com/Gv1rn9Rrcv-3000x3000.png)

```
  BOOL show = CBCentralManagerOptionShowPowerAlertKey;
  if (show != NO) {
```